tools/dimspy/align_samples.xml

```diff
@@ -47,6 +52,22 @@
             <output name="hdf5_file_out" file="pm_as.h5" ftype="h5" compare="sim_size"/>
             <output name="matrix_file_out" file="peak_matrix_as.txt" ftype="tsv"/>
         </test>
+        <test>
+            <param name="hdf5_file_in" value="pls_rf.h5" ftype="h5"/>
+            <param name="ppm" value="2.0"/>
+            <param name="delimiter" value="tab"/>
+            <conditional name="hdf5_to_txt" >
+                <param name="standard" value="True"/>
+                <param name="comprehensive" value="False"/>
+                <param name="samples_representations" value="rows"/>
+                <param name="matrix_attr" value="intensity"/>
+            </conditional>
+            <section name="adv">
+                <param name="block_size" value="10000"/>
+            </section>
+            <output name="hdf5_file_out" file="pm_as.h5" ftype="h5" compare="sim_size"/>
```

Comment:
we can now has_size: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents or special hdf5 asserts like `has_h5_keys` to test for.

Reply:
Thanks for making me aware of these additional asserts features. Very useful!